src/OrbitGl/SourceCodeReport.cpp

```diff
@@ -39,14 +44,23 @@ SourceCodeReport::SourceCodeReport(std::string_view source_file,
       continue;
     }
 
+    min_line_number_ = std::min(min_line_number_, current_line_info.source_line());
+    max_line_number_ = std::max(max_line_number_, current_line_info.source_line());
+
     number_of_samples_per_line_[current_line_info.source_line()] += current_samples;
     total_samples_in_function_ += current_samples;
   }
 }
 
 std::optional<uint32_t> SourceCodeReport::GetNumSamplesAtLine(size_t line) const {
-  if (!number_of_samples_per_line_.contains(line)) return std::nullopt;
-  return number_of_samples_per_line_.at(line);
+  // We are inside the function and we have samples
+  if (number_of_samples_per_line_.contains(line)) return number_of_samples_per_line_.at(line);
```

Comment:
Nit: I don't like the order here.

I would prefer:
- get nullopt checking lines
- this line
- return 0

Reply:
Done.